docs/src/pages/WebNudgersPage/NudgersPage.js

```diff
@@ -53,6 +59,65 @@ class NudgerContainer extends Component<{ buttonType: ?string }, {}> {
   }
 }
 
+const options = ['economy', 'premium', 'business', 'first'];
```

Comment:
Could we write these in title case? I think it'd make for a nicer-looking example (you'd need to change lines, 93, 105 and 106 too).

```suggestion
const options = ['Economy', 'Premium', 'Business', 'First'];
```

Reply:
Done